### PR TITLE
Added SSR contents for new docs

### PR DESCRIPTION
**Pull request by renebrandel**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx [hunk 1]
```diff
@@ -20,18 +20,655 @@ export function getStaticProps(context) {
...
+
+### Step 3 - Call GraphQL API using generated server API clients
+
+With the generated server API clients, you can make any GraphQL query or mutation request. Subscriptions are not available within server runtimes.
+
+<BlockSwitcher>
+
+<Block name="generateServerClientUsingCookies() 🍪">
+
+Import the cookie-based GraphQL API's server client in your Next.js React Server Component code and make your GraphQL.
```
Comment (heatheramz) [inline src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx:129]: @renbran is the apostrophe triggering a code error here?

File: src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx [hunk 2]
```diff
@@ -20,18 +20,655 @@ export function getStaticProps(context) {
...
+
+  return request.data.listTodos.items
+}
+```
+
+</Block>
+
+<Block name="generateServerClientUsingReqRes() 🌐">
+
+Import the NextRequest/NextResponse-based GraphQL API's server client in your Next.js server runtime code and make your GraphQL within the `runWithAmplifyServerContext` function. Review [Server-side Rendering](/[platform]/build-a-backend/server-side-rendering) to learn more about creating an Amplify server context.
```
Comment (heatheramz) [inline src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx:154]: Same error showing here

File: src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx [hunk 3]
```diff
@@ -33,4 +23,391 @@ export function getStaticProps(context) {
...
+
+<Fragments fragments={{ js: versionRangeCallout }} />
+
+Before you begin, you will need:
+- [A Next.js application created](/[platform]/start/project-setup/create-application/)
+- [Installed and configured Amplify libraries for Next.js](/[platform]/build-a-backend/server-side-rendering/)
+
+## Configure Amplify Library for server-side usage
+
+To use Amplify APIs on the server-side of your Next.js app, you will need to
```
Comment (heatheramz) [inline src/pages/[platform]/build-a-backend/server-side-rendering/index.mdx:38]: Doesn't have to be fixed today but this page looks like they have hard breaks for each line which may impact page views on diff screens

**Review by heatheramz — DISMISSED** [review]
Some minor items - I do think you may have an error on the 2 lines I pointed out.

File: src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx (hunk 1 above)
Comment (renebrandel) [reply src/pages/[platform]/build-a-backend/graphqlapi/connect-from-server-runtime/index.mdx:129]: It doesn't and I couldn't understand why it would.

**Review by heatheramz — APPROVED** [review]
LGTM

**Review by hdworld11 — APPROVED** [review]

**Pull request merged**